core/src/main/scala/org/locationtech/rasterframes/expressions/transformers/MaskExpression.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * This software is licensed under the Apache 2 license, quoted below.
+ *
+ * Copyright 2019 Astraea, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     [http://www.apache.org/licenses/LICENSE-2.0]
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.locationtech.rasterframes.expressions.transformers
+
+import geotrellis.raster.Tile
+import org.apache.spark.sql.catalyst.analysis.TypeCheckResult
+import org.apache.spark.sql.catalyst.analysis.TypeCheckResult.{TypeCheckFailure, TypeCheckSuccess}
+import org.apache.spark.sql.catalyst.expressions.Expression
+import org.apache.spark.sql.types.DataType
+import org.locationtech.rasterframes.expressions.DynamicExtractors.tileExtractor
+
+import spire.syntax.cfor._
+
+trait MaskExpression { self: Expression =>
```

Comment:
We can cut a fresh GT release with the fixed mask operation! Sadly due to the CircleCI breach the most up to date snapshot has not been released.

Reply:
Actually this is kind of fine. The logic here is slightly different from the standard implementation because we know we're mapping over int values. I would bet that it would be more helpful in the future to have full control over this than saving a few lines of code.